probables/cuckoo/cuckoo.py

```diff
@@ -337,13 +354,14 @@ def export(self, file: Union[Path, str, IOBase, mmap]) -> None:
                 bucket = self.buckets[i]
                 # do something for each...
                 if isinstance(bucket, list):
-                    self.buckets[i] = bucket = convert_to_typed(self.__class__.SINGLE_INT_C, bucket)  # type: ignore
+                    a = convert_to_typed(self.SINGLE_INT_C, bucket)
+                    self.buckets[i] = a.tolist()
```

Comment:
Does using a typed array internally cause any issues?

Reply:
I ***believe*** that would be ok but I want to tackle that in another PR. Specifically to address issue #60 as I think it could be done for several of the classes underlying data.

Reply:
As soon as this PR is approved and merged, I will begin tackling that; when that is done then I will likely cut a new release.